template-for-repository/proofs/Makefile.common

```diff
@@ -433,6 +433,7 @@ veryclean: clean
 
 JOB_OS ?= ubuntu16
 JOB_MEMORY ?= 32000
+EXPECTED ?= SUCCESSFUL
```

Comment:
Please write a comment here. Suggest something like
```
Proofs that are expected to fail should set EXPECTED to
"FAILURE" in their Makefile. Values other than SUCCESSFUL
or FAILURE will cause a CI error.
```

Reply:
Thanks for the quick feedback. Done.

Reply:
@danielsn @feliperodri I think this should actually be `FAILED` rather than `FAILURE`, right? My mistake.

Reply:
Yep, CBMC reports VERIFICATION FAILED.

Reply:
Thanks, made then change.